scheduling-api-http/src/main/resources/application-test.properties

```diff
@@ -0,0 +1,108 @@
+#
+# ProActive Parallel Suite(TM):
+# The Java(TM) library for Parallel, Distributed,
+# Multi-Core Computing for Enterprise Grids & Clouds
+#
+# Copyright (c) 2016 ActiveEon
+# Contact: contact@activeeon.com
+#
+# This library is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License
+# as published by the Free Software Foundation: version 3 of
+# the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# If needed, contact us to obtain a release under GPL Version 2 or 3
+# or a different license than the AGPL.
+#
+
+# Configure logging level
+logging.level.org.ow2.proactive.scheduling.api=debug
+logging.level.org.springframework=off
+logging.level.org.hibernate.SQL=off
+# Embedded server configuration
+server.compression.enabled=false
+server.contextPath=/
+server.port=9999
+# Disable Spring banner
+spring.main.banner_mode=off
+# Disable Spring security
+security.basic.enabled=false
+management.security.enabled=false
+
+##############
+# DATASOURCE #
+##############
+
+# The default settings are using hsqldb
+######################## HSQL DB #######################################################
+spring.datasource.driverClassName=org.hsqldb.jdbc.JDBCDriver
+spring.datasource.url=jdbc:hsqldb:mem:scheduler
+spring.jpa.database-platform=org.hibernate.dialect.HSQLDialect
+
+spring.datasource.username=root
+spring.datasource.password=
+
+######################## MYSQL DB ########################################################
```

Comment:
Maybe it's easier to use commented out configurations for various database, but why not use different files instead ?

Reply:
same answer as for the catalog